spiking_neurons/spiking_neuron.jl

```diff
@@ -0,0 +1,77 @@
+using Revise, MathematicalSystems,Reachability, LinearAlgebra, HybridSystems
+using LazySets;
+using Reachability: solve
+using Plots
+using TaylorIntegration
+
+@taylorize function spiking_neuron!(t, x, dx)
+    local a = 0.02
+    local b = 0.2
+    local I = 40
+    dx[1] = 0.04*x[1]^2 + 5*x[1] + 140 - x[2] + I
+    dx[2] = a*(b*x[1] - x[2])
```

Comment:
IIRC (but I might be wrong), there are performance issues with nested multiplications. Maybe the following is better:
```suggestion
    dx[2] = a*b*x[1] - a*x[2]
```
@mforets: Do you know?

Reply:
use binary operations with `tayorize`, so i think that here `a*(b*x[1] - x[2])` is ok

Reply:
however, the differential equation for `x[1]` should be rewritten eg.

```julia
dx[1] = (0.04*(x[1]*x[1]) + 5*x[1]) + (140+I-x[2])
```

Reply:
With regards to using `@taylorize`, Marcelo's solution is ok; another option is
```julia
dx[1] = (0.04*x[1]^2) + 5*x[1]) + ((140+I)-x[2])
```
The important thing is that you use parenthesis when more than two operations/functions are involved. This restriction is related to the way the internals of `@taylorize` work.

Reply:
One parenthesis is unmatched.